sample-kotlin/src/test/java/com/infullmobile/android/infullmvp/sample/sample_mvp_card/SampleMvpCardTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.infullmobile.android.infullmvp.sample.sample_mvp_card
+
+import com.infullmobile.android.infullmvp.basetest.InFullMvpCardBaseTest
+import com.infullmobile.android.infullmvp.sample.activity.SampleActivity
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.di.SampleMvpCardModule
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.di.SampleMvpCardScope
+import dagger.Provides
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnit
+import org.mockito.junit.MockitoRule
+import org.robolectric.RobolectricTestRunner
+import org.mockito.Mockito.`when` as whenDo
+
+@RunWith(RobolectricTestRunner::class)
+class SampleMvpCardTest : InFullMvpCardBaseTest<
+        SampleMvpCard,
+        SampleMvpCardPresenter,
+        SampleMvpCardView,
+        SampleActivity>() {
+
+    @get:Rule val rule: MockitoRule = MockitoJUnit.rule()
+
+    @Mock lateinit var mockedPresenter: SampleMvpCardPresenter
+    override val activityClass = SampleActivity::class.java
```

Comment:
redundant

Reply:
still reduntant, please remove

Reply:
we need to pass this `class.java` thing to robolectric, and we cant do it on generic (kotlin says it would need to be reified, but it cant because only inlined functions can...)